rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/270_median_absolute_deviation_metric.yml

```diff
@@ -0,0 +1,122 @@
+setup:
```

Comment:
Could we add a test where there are no docs (or no fields) that match, so we can verify the xcontent spits out a `null` value when nothing was collected?

Reply:
Yeah this is definitely important, thanks for catching